src/tyr/trace_serializer.cc

```diff
@@ -19,6 +19,9 @@ constexpr size_t kConfidenceScoreIndex = 0;
 constexpr size_t kRawScoreIndex = 1;
 constexpr size_t kMatchResultsIndex = 2;
 constexpr size_t kTripLegIndex = 3;
+constexpr double MILLISECOND_TO_SEC = 0.001;
```

Comment:
we have similar constants in midgard/constants.h move missing ones there. feel free to also make double versions of these if it suits your fancy